engine/render/src/render/render_script.cpp

```diff
@@ -3110,6 +3180,8 @@ namespace dmRender
         {"constant_buffer",                 RenderScript_ConstantBuffer},
         {"enable_material",                 RenderScript_EnableMaterial},
         {"disable_material",                RenderScript_DisableMaterial},
+        {"set_compute_program",             RenderScript_SetComputeProgram},
+        {"dispatch",                        RenderScript_Dispatch},
```

Comment:
maybe this should be called "dispatch_compute"?

Reply:
I think it might be nice for discoverability/grouping of the concepts.